tests/utils.py

```diff
@@ -206,13 +207,11 @@ def make_java_profiler(
     profiling_mode: str = CPU_PROFILING_MODE,
 ) -> JavaProfiler:
     assert storage_dir is not None
+    profiler_state = ProfilerState(stop_event, storage_dir, profile_spawned_processes, insert_dso_name, profiling_mode)
```

Comment:
Use the same `make_profiler_state` function that I suggested here - https://github.com/Granulate/gprofiler/pull/668/files#r1096248108

Ideally, the interface of `make_java_profiler` will have `ProfilerState` as well, and not the old arguments, but I assume that'll require a massive change in the tests? Please try it out, if it's a small change then let's fix it here as well to keep a consistent API of `make_..._profiler`. If it's a big change, we can skip, just make sure you use the same `make_profiler_state`.